API/ev3_timer.c

```diff
@@ -83,9 +82,7 @@ void TimerReady(unsigned long Timer)
   // wait until timer has elapsed
   // do not block other threads
   while (Timer > TimerGetMS())
-  {
-	usleep(1000);
-  }
+    Wait(1);

```

Comment:
I don't see the benefit. If you get interrupted, you will still remain in the loop, so adding a second loop doesn't do much.

Reply:
Replaced with b3b6b867405554eab66e805d7a7404bbe0c7f896 (I kept the include change in the case that there is also some other code that needs it) :+1: